app/src/main/java/edu/gatech/cs2340/hkskh/Shelters/ShelterServiceProvider.java

```diff
@@ -0,0 +1,73 @@
+package edu.gatech.cs2340.hkskh.Shelters;
+
+import android.content.Context;
+
+import com.opencsv.CSVReader;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import edu.gatech.cs2340.hkskh.R;
+import edu.gatech.cs2340.hkskh.Shelters.Models.Shelter;
+
+/**
+ * Created by henry on 2/27/18.
+ */
+public class ShelterServiceProvider {
+
+    /**
+     * Load data from CSV file
+     */
+    public static void load(Context context) {
+        // Load from CSV
+        try {
+            InputStream is = context.getResources().openRawResource(R.raw.shelters);
+            CSVReader reader = new CSVReader(new BufferedReader(new InputStreamReader(is, "UTF-8")));
+            String[] row;
+            reader.readNext(); // Dump header
+            while ((row = reader.readNext()) != null) {
+                try {
+                    int key = Integer.parseInt(row[0]);
+                    String name = row[1];
+                    String capacityInd = row[2]; // TODO parse out individual and family
```

Comment:
Couple TODOs here, but I didn't have time to perfect it. Wanted to let Kirby get started.